manager/loginmanager/login.go

```diff
@@ -25,7 +24,7 @@ func LocalLoginUser(userStore *store.UserStore, accessGenerate *generates.JWTAcc
 		return nil, err
 	}
 
-	storedUser, err := userStore.GetUser(bson.M{"username": username, "kind": models.LocalAuth})
+	storedUser, err := usermanager.GetUser(userStore, bson.M{"username": username, "kind": models.LocalAuth})
```

Comment:
Why this change? Please describe.

Reply:
Wrote a function in `usermanager` which will be sole responsible for talking to `userstore` in any get query. All the functions that need to get a user will have to use `usermanager` only